docs/data-routing/4-destinations/14-lightdb-state.md

```diff
@@ -0,0 +1,60 @@
+---
+title: lightdb-state
+---
+
+|   |   |
+|---|:---:|
+|__Latest Version__| `v1.0.0` |
+|__Input Content Type__| `application/json` |
+
+:::usage
+Sending data to LightDB State incurs usage costs after exceeding the free tier.
+See [Golioth pricing](https://golioth.io/pricing) for more information.
+:::
+
+The LightDB State destination sends data to [LightDB
+State](/application-services/lightdb-state). Data must have arrived as or been
```

Comment:
```suggestion
State](/application-services/lightdb-state). Data must have arrived as or be
```

Reply:
Thanks! I'll remove `have` and change `arrived` -> `arrive` as well 👍🏻